### ISSUE TITLE
Possible to support specific deploy targets?

### ISSUE THREAD

**Issue by jhuleatt** [FirebaseExtended/action-hosting-deploy#6]

**Comment by jhuleatt** [comment]
After some testing, the CLI supports this like this:

```
firebase hosting:channel:deploy --only <target_name_1>,<target_name_2> <channel_id>
```

**Comment by expatiating** [comment]
@jhuleatt this would be a very useful feature. 

I currently have a Firebase Project (`my-project`) with Hosting configured for 2 sites (`my-site-1`, `my-site-2`).
When I build and deploy `my-project` on the command line I run this 
```
firebase deploy --only hosting:my-target-1 --project my-project
```
My `.firebaserc` is like this:
```
{
  "projects": {
    "default": "my-project"
  },
  "targets": {
    "my-project": {
      "hosting": {
        "my-target-1": [
          "my-site-1"
        ],
        "my-target-2": [
          "my-site-2"
        ]
      }
    }
  }
}
```
I can imagine having a `target` property on the `with` property under `uses` such as 
```
      - uses: FirebaseExtended/action-hosting-deploy@v0
        with:
          repoToken: "${{ secrets.GITHUB_TOKEN }}"
          firebaseServiceAccount: "${{ secrets.FIREBASE_SERVICE_ACCOUNT }}"
          channelId: live
          target: my-target-1
```

I'd be happy to help with this feature, but still studying your code. I see a potential place to hook in on `L21`
https://github.com/FirebaseExtended/action-hosting-deploy/blob/79d1e17528573ce1a02632923c928f62009df3c3/src/deploy.ts#L19-L24

but not yet sure how to thread that through. Will keep investigating.

**Comment by jhuleatt** [comment]
fixed by #41
